src/pages/index.js

```diff
@@ -1,37 +1,178 @@
+);
+const addPlacePopup = new PopupWithForm("#add-card-modal", handleAddCardSubmit);
+
+const profilePicPopup = new PopupWithForm(
+  "#edit-pic-modal",
+  handleEditPicSubmit
+);
+
+const confirmDeletePopup = new PopupWithForm(
+  "#confirm-modal",
+  handleDeleteCard
+);
+
+//User Info Class
+const userInfo = new UserInfo({
+  nameSelector: ".profile__name",
+  jobSelector: ".profile__subtitle",
+});
 
-const { formSelector } = config;
+//Form Validator Class
+const editProfileFormValidator = new FormValidator(
+  config,
+  editProfilePopup.popupForm
+);
+
+const addPlaceFormValidator = new FormValidator(
+  config,
+  addPlacePopup.popupForm
+);
+
+const profilePicFormValidator = new FormValidator(
+  config,
+  profilePicPopup.popupForm
+);
 
-//Button Elements
+//Open Buttons
 
-const editProfileMOB = document.querySelector("#edit-profile-modal-open-button");
+const editProfileMOB = document.querySelector(
+  "#edit-profile-modal-open-button"
+);
 const addCardMOB = document.querySelector("#add-card-modal-open-button");
+const profilePic = document.querySelector("#profile-photo");
+
+//Form Placeholders
 const userNameInput = document.querySelector("#edit-profile-name-placeholder");
-const userJobInput = document.querySelector("#edit-profile-subtitle-placeholder");
+const userJobInput = document.querySelector(
+  "#edit-profile-subtitle-placeholder"
+);
+
+//User Info
+const profileName = document.querySelector("#profile-name");
+const profileAbout = document.querySelector("#profile-subtitle");
+
+//Modals
+const editProfilePicModal = document.querySelector("#edit-pic-modal");
+const addCardModal = document.querySelector("#add-card-modal");
+const editProfileModal = document.querySelector("#edit-profile-modal");
 
+//Popups
+const popups = [
+  maxImagePopup,
+  editProfilePopup,
+  addPlacePopup,
+  profilePicPopup,
+  confirmDeletePopup,
+];
+
+//Form Validators
+const formValidators = [
+  editProfileFormValidator,
+  addPlaceFormValidator,
+  profilePicFormValidator,
+];
+
+//Card Container
+let cardSection;
 
 //***************************************************************************************************************************//
 //                                                 Functions                                                                 //
 //***************************************************************************************************************************//
 
-const createCard = (cardData, method = "append") => {
-  const cardTemplate = new Card(cardData, "#card-template", handleImageClick);
+//Creates new Cards
+function addNewCard(cardData, method = "append") {
+  const cardTemplate = new Card(
+    cardData,
+    "#card-template",
+    handleImageClick,
+    handleConfirmDelete,
+    handleLikeCard
+  );
   const cardElement = cardTemplate.getCardElement();
+  cardTemplate.renderLikes();
 
   cardSection.addItem(cardElement, method);
-};
+}
+
+//Sets Profile Picture
+async function setProfilePic() {
```

Comment:
Please, delete `setProfilePic()` because you call the same `api.getUserInfo()` inside `setUserInfo` in line 151, so you need to set the avatar there as well